### ISSUE TITLE
cat

### ISSUE THREAD

**Issue by masaisai111** [tencent-ailab/IP-Adapter#345]
![Snipaste_2024-04-24_18-43-05](https://github.com/tencent-ailab/IP-Adapter/assets/109290222/ef616d76-4031-436b-ac89-d285d9ede1fe)
For this formula, why is prompt_embeds = torch.cat([prompt_embeds_, image_prompt_embeds], dim=1) reflected in the code?

**Comment by xiaohu2015** [comment]
https://github.com/tencent-ailab/IP-Adapter/blob/main/ip_adapter/attention_processor.py#L82
